src/components/tab-panel/tab-panel.scss

```diff
@@ -1,15 +1,20 @@
-:host(limel-tab-panel) {
-    display: block;
-}
+@import '../../style/internal/variables';
 
-.tab-panel {
+.tab-panel { // contains both limel-tab-bar and tab-content
     height: 100%;
+    position: relative;
     display: flex;
     flex-direction: column;
 }
 
 .tab-content {
-    height: 100%;
-    overflow: auto;
-    flex: 1;
+    position: absolute;
```

Comment:
Just a small question in order for me to understand and learn :slightly_smiling_face: 
What is the benefit of doing it this way with absolute positioning?

Reply:
well, the short answer is: "That was the only way I could make it work while throwing things around in different places." 🙃
But now when I test with `position: relative` and `flex` and so forth, that works too. Probably something else we did solved the issues. So there is no point in having absolute position anymore. So I change this too.